File: mongodb/models/Driver.js
import mongoose  from "mongoose"

const DriverSchema = new mongoose.Schema({
    firstName: { type: String, required: true}, 
    lastName: { type: String, required: true}, 
    licenseNumber: { type: String, required: true }, 
    dateOfBirth: { type: String, required: true }, 
    phoneNumber: { type: String, required: true},
    email: { type:String, required: true}, 
    vehicle: {type:String}
})

const Driver = mongoose.model('Driver', DriverSchema)

export default Driver

File: app/src/main/java/com/example/test/models/Driver.kt
package com.example.test.models

import java.io.Serializable


data class Driver(
    val _id: String?,
    val userId: String?,
    val firstName: String?,
    val lastName: String?,
    val licenseNumber: String?,
    val licenseExpiry: String?,
    val dateOfBirth: String?,
    val phoneNumber: String?,
    val email: String?,
    val imageUrl: String?,
    val vehicle: VehicleAdd?
    )








